pinot-common/src/main/java/org/apache/pinot/common/function/scalar/StringFunctions.java

```diff
@@ -560,4 +561,26 @@ public static String decodeUrl(String input)
       throws UnsupportedEncodingException {
     return URLDecoder.decode(input, StandardCharsets.UTF_8.toString());
   }
+
+  /**
+   *
+   * @param input utf-8 encoded String
```

Comment:
Java String object is not UTF-8. This should probably just say `@param input input string`.

Reply:
Resolved - updated implementation passes byte[] as argument instead of String.